src/components/Tree/Node.tsx

```diff
@@ -53,6 +53,31 @@ type NodeProps = {
     onEditableSave?: (targetItemId: string, value: string) => void;
 };
 
+const setShowNodesInitial = (currentNode: DraggableItem<TreeNodeItem>, activeNodeIds?: NullableString[]): boolean => {
```

Comment:
Would it be possible to find another name for this function? I think maybe en `setShowNodesOnInit` would be more semantic.

Reply:
What about `getShowNodesOnIt`? Now that I look at it, this function doesn't set a value, but gets one.

Reply:
Renamed to `getInitialShowNodesValue`, as I realized it's not a setter, but rather a getter. And it seems descriptive, as it is getting the initial value for the `showNodes` state variable. But happy to rename to anything else